packages/lodestar/src/chain/stateTransition/util/attestation.ts

```diff
@@ -117,10 +116,8 @@ export function getAttestingIndices(
   bits: BitList
 ): ValidatorIndex[] {
   const committee = getCrosslinkCommittee(config, state, data.target.epoch, data.crosslink.shard);
-  assert(bits.bitLength === committee.length);
```

Comment:
why is this removed?

Reply:
It's not in spec:
https://github.com/ethereum/eth2.0-specs/blob/v0.8.3/specs/core/0_beacon-chain.md#get_attesting_indices

Reply:
ok, looks like we're missing the length check here:
 in https://github.com/ethereum/eth2.0-specs/blob/v0.8.3/specs/core/0_beacon-chain.md#attestations